src/lib/logging.ts

```diff
@@ -0,0 +1,40 @@
+import { ClientsController, ZwavejsServer } from "./server";
+
+export class LoggingEventForwarder {
+  /**
+   * Only load this once the driver is ready.
+   *
+   * @param clients
+   * @param server
+   */
+  public started: boolean = false;
+
+  constructor(
+    private clients: ClientsController,
+    private server: ZwavejsServer
+  ) {}
+
+  // Forward logging events on to clients that are currently
+  // receiving logs
+  forwardLogging(message: string) {
+    this.clients.clients
```

Comment:
I keep getting the following error here and I can't figure out why:

TypeError: Cannot read property 'clients' of undefined
    at ZwavejsServer.forwardLogging (/usr/src/app/node_modules/@zwave-js/server/dist/lib/logging.js:19:22)
    at ZwavejsServer.emit (node:events:378:20)
    at ZwavejsServer.EventEmitter.emit (node:domain:470:12)
    at EventEmitterLogTransport.log (/usr/src/app/node_modules/@zwave-js/server/dist/lib/server.js:296:22)
    at EventEmitterLogTransport._write (/usr/src/app/node_modules/winston-transport/index.js:103:17)
    at doWrite (/usr/src/app/node_modules/readable-stream/lib/_stream_writable.js:428:64)
    at writeOrBuffer (/usr/src/app/node_modules/readable-stream/lib/_stream_writable.js:417:5)
    at EventEmitterLogTransport.Writable.write (/usr/src/app/node_modules/readable-stream/lib/_stream_writable.js:334:11)
    at DerivedLogger.ondata (/usr/src/app/node_modules/winston/node_modules/readable-stream/lib/_stream_readable.js:681:20)
    at DerivedLogger.emit (node:events:390:22)

Reply:
I fixed this by inlining the function instead of having a separate one. The only potential problem with that is that the only way to remove the listener is to remove all listeners. Right now, this isn't a problem but it could be in the future if we decide to listen to the events elsewhere

Reply:
Ah, the beauty of `this` in JavaScript. 

The easiest is to force this function to be bound to the instance by storing a lambda as a public var.
```ts
public log = (info: ZWaveLogInfo, next: () => void): void => {
```

Reply:
this is no longer a concern because we are doing the forwarding to the client within the LogTransport itself, which I think is a better solution anyway since we don't have to add redundant overhead to the event bus. But good to know for the next time 😓